plugin/pkg/skplug/grpc.go

```diff
@@ -65,6 +65,17 @@ func (m *GRPCClient) Scale(partition string, time int64) (rec int32, err error)
 	return resp.Rec, nil
 }
 
+func (m *GRPCClient) VerticalRecommendation(partition string, time int64) (rec []*proto.RecommendedPodResources, err error) {
```

Comment:
I would expect the VPA plugin to be in a different go module / binary from the HPA plugin.  Since they are different repositories, so we can manage their dependency graphs separately.

Reply:
Nevermind.  You are modifying the boilerplate golang code that goes with the plugin proto that each individual plugin (HPA and VPA will import).